packaging/nnstreamer.spec

```diff
@@ -34,7 +34,8 @@ BuildRequires:  pkgconfig(libpng)
 BuildRequires: tensorflow-lite-devel
 
 %if 0%{?testcoverage}
-BuildRequires:	taos-ci-unittest-coverage-assessment
```

Comment:
It's reasonable to me at least because we can not use the existing OBS server to the the `taos-ci-unittest-coverage-assessment` package due to the firewall.